docs/src/examples/components/Chat/Content/ChatExampleReactionGroup.shorthand.tsx

```diff
@@ -10,7 +10,7 @@ const items = [
     message: {
       content: (
         <Chat.Message
-          reactionGroup={{ items: reactions }}
+          reactions={reactions}
```

Comment:
Breaking change?

Reply:
I added this as a showcase... Maybe will create different PR for changing this later...

Reply:
yes, lets introduce changes to simplify all the related component APIs as a separate PR - however, probably, it is better to do it sooner than later :)

Reply:
Will revert this changes from this PR, is getting pretty big as it is..